### DISCUSSION TITLE
Recode hive badge in organization

### DISCUSSION THREAD

**Discussion by Sameeratweb** [recodehive/Stackoverflow-Analysis#447] [Q&A] [answered]
is it possible to get recodehive badge under organization section? my pr's are merged too.

**Comment by sanjay-kv** [comment]
yes once you joined.. 
got to people, search your name. make badge to public

**Comment by Sameeratweb** [comment, marked as answer]
ive checked my name is not present

**Comment by Sameeratweb** [comment]
how can i join?

**Reply by sanjay-kv** [reply]
https://github.com/Recode-Hive/Support/issues/new?assignees=&labels=invite+me+to+the+community&projects=&template=invitation.yml&title=Please+invite+me+to+the+Recode-Hive+GitHub+Community+Organization

**Reply by Sameeratweb** [reply]
Thankyou

**Comment by RaviSingh733** [comment]
Thank You

**Comment by OrasanuAna** [comment]
Yes, it’s possible to get the RecodeHive badge under the organization section, especially if your pull requests have already been merged. I recommend reaching out to the organization admins to ensure the badge is granted correctly.

**Comment by sanjay-kv** [comment]
@Sameeratweb 
watch the video below. 

https://www.loom.com/share/42b96ebe361e4c75a60d44c8eede2b83?sid=a56d7e0b-2508-4f91-9de2-6e87d7163537

**Comment by TatyOko28** [comment]
Hi,
If your PRs are merged, that's awesome! 🎉 You might want to check whether the Recode Hive badge supports organizational accounts or if it's limited to individual contributors.

Try reaching out to the Recode Hive team directly or checking their GitHub discussions—they might have more details on this! 🚀
